beacon-chain/rpc/eth/beacon/validator.go

```diff
@@ -298,7 +299,7 @@ func valContainersByRequestIds(state state.BeaconState, validatorIds [][]byte) (
 			valContainers = append(valContainers, &ethpb.ValidatorContainer{
 				Index:     valIndex,
 				Balance:   allBalances[valIndex],
-				Status:    subStatus,
+				Status:    ethpb.ValidatorStatus(uint8(subStatus)),
```

Comment:
actually it turns out it's not because the type of `subStatus` is an alias for `int8` and the type of `Status` is an alias for `int32`, and therefore it's directly assignable.